doc/etc/config.ini

```diff
@@ -36,10 +36,17 @@ path=/srv/dlsnode/dlsnode.pid
 ;
 ; write_flush_ms        millisecond interval at which to flush the buffers of
 ;                       any stream requests
+; file_buffer_size      Size of the buffer used for buffering file reads.
```

Comment:
optional: you might want to clarify the size of the buffer is in bytes

Reply:
Updated.